### PR TITLE
feat: integration with notebooksd

### PR DESCRIPTION
**Pull request by genehynson**
https://github.com/influxdata/idpe/issues/9806

Adds integration with `notebooksd` behind an unofficial feature flag called `notebooks-api`.

- adds `notebooks.yml` swagger
- adds `notebooks.api.tsx` which is identical to `api.tsx` for flows, but uses `notebooks.yml` swagger

To use this flag you'll first need to enable `notebooksd` in k8s-idpe locally by flipping this flag: https://github.com/influxdata/k8s-idpe/blob/3f0b5eb24b144802b62e425caae1dfd7706993a7/env/test/local/twodotoh-dev/conf.libsonnet#L221

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 59 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=1283)

File: src/flows/context/notebooks.api.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    throw new Error(res.data.message)
+  }
+}
+
+export const createAPI = async (flow: PostNotebookParams) => {
+  const res = await postNotebook(flow)
+  if (res.status != 200) {
+    throw new Error(res.data.message)
+  }
+  return res.data.id
```
Comment (drdelambre) [inline src/flows/context/notebooks.api.tsx:68]: can we get rid of all of this duplication and handholding once we have our first migration?

**Review by drdelambre — APPROVED** [review]
this is a lot of code just to change a string on the fetch request

**Comment by ui-ci-bot-influx** [comment]
> this is a lot of code just to change a string on the fetch request

Yeah but all of the duplication will be removed after we switch over. This seemed like the cleanest way for me to separate the two code paths.

**Comment by genehynson** [comment]
> this is a lot of code just to change a string on the fetch request

Yeah but most of the duplication will be removed after we switch over. This seemed like the cleanest way for me to separate the two code paths.

File: src/flows/context/notebooks.api.tsx (hunk 1 above)
Comment (genehynson) [reply src/flows/context/notebooks.api.tsx:68]: yes!

**Pull request merged**